Serilog.Sinks.Network/NetworkLoggerConfigurationExtensions.cs

```diff
@@ -41,37 +41,46 @@ public static LoggerConfiguration TCPSink(
             this LoggerSinkConfiguration loggerConfiguration,
             IPAddress ipAddress,
             int port,
-            ITextFormatter textFormatter = null,
+            int? writeTimeoutMs = null,

```

Comment:
we could have a signature overload without these that provides defaults so that it isn't a breaking change for existing users

(that said, i've no idea if there _are_ existing users and a breaking change is fine too, since we're also bumping serilog version so 🤷 )